container-search-gui/src/main/resources/gui/_includes/css/vespa.css

```diff
@@ -0,0 +1,626 @@
+:root{
```

Comment:
Missing copyright. Did you write it from scratch? Otherwise state what open source code it is based on.

Reply:
It is based on the vespa.css in frontpage-repo, it has the styling for header and footer.